### PR TITLE
114 health check route

### LINKED ISSUE #114: health check route 추가

### PR DESCRIPTION
**Pull request by BLoHny**
# summary
- health check 추가
# changes
- db, redis health check route 
- entgo client 구조 수정

close #114

### CODE REVIEW

File: cmd/app/main.go [hunk 1]
```diff
@@ -105,7 +109,7 @@ func main() {
 	defer closeRedis()
 
 	mysqlConf := config.Data().Mysql
-	ent, closeMysql, err := ent.NewClient(ent.NewMySQLDialect(ent.MysqlDialectOpts{
+	ent, drv, closeMysql, err := ent.NewClient(ent.NewMySQLDialect(ent.MysqlDialectOpts{
```
Comment (onee-only) [inline cmd/app/main.go]: drv보단 db나 sqlDB가 나을 것 같습니다!

File: cmd/app/main.go [hunk 2]
```diff
@@ -192,6 +196,9 @@ func main() {
 	logHandler := interceptor.NewLogger(logger)
 	ratelimiter := middleware.NewRateLimiter()
 
+	sqlCheck := checks.SqlCheck{Sql: drv}
+	redisCheck := checks.NewRedisCheck(redis)
```
Comment (onee-only) [inline cmd/app/main.go]: 두 check을 같은 방법으로 initialize 시킬 수 없을까요?

Comment (BLoHny) [reply cmd/app/main.go]: https://github.com/lucky-pocket/luckyPocket-back/pull/117/commits/12b2f7e50112e10567033da20e6d1740dbf1b2f0

File: cmd/app/main.go (hunk 1 above)
Comment (BLoHny) [reply cmd/app/main.go]: https://github.com/lucky-pocket/luckyPocket-back/pull/117/commits/c3f8122306696a276c499d24bef5a3a801e75738

**Review by onee-only — APPROVED** [review]

**Pull request merged**